docs/sce_file_contents.md

```diff
@@ -53,6 +53,12 @@ These metrics were calculated by using [`miQC`](https://bioconductor.org/package
 | `prob_compromised`      | Probability that a cell is compromised (i.e., dead or damaged), as calculated by `miQC`                                                                                                       |
 | `miQC_pass`             | Indicates whether the cell passed the default miQC filtering. `TRUE` is assigned to cells with a low probability of being compromised (`prob_compromised` < 0.75) or [sufficiently low mitochondrial content](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html#preventing-exclusion-of-low-mito-cells).  |
 
+For the `_processed.rds` file only, an additional column named, `ccdl_filter` will be included in the `colData`.
```

Comment:
Did we decide not to add this back to the filtered data? I thought that we had, according to: 
https://github.com/AlexsLemonade/scpca-nf/blob/f797157dff44804500774f164cc518017c0871e8/bin/post_process_sce.R#L147-L148
and 
https://github.com/AlexsLemonade/scpca-nf/blob/f797157dff44804500774f164cc518017c0871e8/modules/sce-processing.nf#L126-L134

Reply:
Yes you are right! I'll update this.